### PR TITLE
from bootstrap to vuetify

### PR DESCRIPTION
**Pull request by DenizCankurtaran**
I completely got rid of bootstrap, every View only uses vuetify.
But I had to make some minor changes, because some components in bootstrap don't exist in vuetify.

In the Generator.vue each input field had a label over the input, vuetify has a feature that combines label and input into one component, I used that through each Generator Component, because I think it's neat, but we can also go back to the old design.
Because of this feature the Select inputs no longer display 'Choose...', instead they display the label, a minor problem is that 'Choose...' had a default value, of maybe 2, so the user didn't have to necessarily change the value and could just leave it as 'Choose...', that is no longer possible and the user *has* to choose a value, but we could change it back.

The Buttons displayed in Welcome used to be bigger, but vuetify can't directly change the Button size through css, so they are a bit smaller.
They also don't work very well on smaller devices, we could change them to div's or any other clickable field.

One slightly bigger problem was the accordion displayed in the Profile, that showed how to download the config file.
Vuetify doesn't have an accordion, but they have something really similar.
Instead of the accordion I used vuetifys expansion-panel.
But currently there is a problem and I don't know how to fix it.

Screenshot from 2020-11-17 12-04-20

When you click on the Panel this weird border shows and I don't have the slightest idea why.
I've tried to fix it, but I didn't understand why it has this form in the first place.

### CODE REVIEW

File: src/components/KubeConfig/KubeInstallation.vue [hunk 1]
```diff
@@ -64,16 +53,11 @@ export default {
...
-  }
-  .btn:hover {
-    background-color: #8BC34A !important;
-  }
-  .icon {
-    color: #F1F8E9;
+  .accordion{
+    width: 60vw;
+    margin: 0 auto;
+    //$expansion-panel-border-radius: 0px !important
```
Comment (DenizCankurtaran) [inline src/components/KubeConfig/KubeInstallation.vue:59]: This is the potential fix I was talking about, but it doesn't work. I've also added a link to where vuetify explains custom variables.

**Comment by kat-hi** [comment]
funny bug. I fixed it with using an additional div that now includes the v-expansion-panels.

**Review by kat-hi — CHANGES_REQUESTED** [review]
thanks, great. :)
 I don't know if you're still on it but I would like to request minor css fixes or changes (as you want): the webapps in the welcome view, the button style in profile and the generator margins. you are free to change the style but it is enough to hotfix the current css.

**Comment by DenizCankurtaran** [comment]
I have changed the requested styles.

But in the code, I had to make something rather "ugly", since each component has scoped style, each generator component has now a generator_form css class which does exactly the same.
I wondered if we could have a .vue file which doesn't have scope styling and contains our main styles.

For example Kubeportal.vue, if the style were not scoped, we could dump all of our css, which multiple components use, in there.

**Review by kat-hi — APPROVED** [review]

**Comment by kat-hi** [comment]
I like your idea. I've already started it in index.html (/public) but never used this concept for components because you already have the component-concept that protect you from repeating yourself.

so we have 2 possibilities: change the three generator components and create only one generic generator component that can be used as ingress, deployment or service. Or we try your idea. and I think yours is much easier in that case, so go for it.
I suggest to use this unscoped css only if it's really necessary.

you can do that in another branch and merge this one :)

**Pull request merged**